### PR TITLE
AS-186: swapped out call boto_init.sh which was causing job failures …

### PR DESCRIPTION
**Pull request by wjdonnelly**
…in staging and prod

https://amplify-litco.atlassian.net/browse/AS-186

### CODE REVIEW

**Comment by pchand-amplify** [comment]
version bump.

File: jenkins/disco_app_conf_update.sh [hunk 1]
```diff
@@ -14,7 +14,7 @@ if [[ "$1" == "-h" || "$1" == "--help" || "$1" == "" ]] ; then
     exit 1
 fi
 
-source "$(dirname $0)/boto_init.sh"
+source /opt/wgen/bin/use_asiaq_from_rpm.sh
```
Comment (pchand-amplify) [inline jenkins/disco_app_conf_update.sh]: I think we already do this here https://github.com/amplifylitco/asiaq_astro_config/blob/8ae71b8f8afb7208d35014f4360f26f8599d6c72/discoroot/etc/init.d/disco-jenkins~for_deployenator#L146 .

Comment (wjdonnelly) [inline jenkins/disco_app_conf_update.sh]: So i'll just remove the line since it is redundant

**Comment by jconstance-amplify** [comment]
Did we ultimately decide to not fix this this way and we should close this PR?

**Comment by pchand-amplify** [comment]
Am closing this PR, since the bug was somewhere else.

**Pull request closed without merging**